sentry-android-ndk/src/main/java/io/sentry/android/ndk/IModuleListLoader.java

```diff
@@ -0,0 +1,10 @@
+package io.sentry.android.ndk;
+
+import io.sentry.protocol.DebugImage;
+
+/** Used only for making the Module list loader testable */
+interface IModuleListLoader {
```

Comment:
`l`: If we only need this for testing, we could also remove the interface. On the JVM we can also mock final classes.

Reply:
makes sense, unnecessary abstraction, thanks